### ISSUE TITLE
PostgreSQL: function parameters in DDL are incorrectly prompted to be bound on script execution

### ISSUE THREAD

**Issue by MarcoBorchert** [dbeaver/dbeaver#7726]
DBeaver 6.3.3
Postgres 9

Steps to reproduce:
 - Enable **Enable SQL parameters** in **SQL Processing**.
 - Try to execute following script:

```
DO LANGUAGE plpgsql 
$$
BEGIN
	CREATE FUNCTION foo(integer) RETURNS integer
    AS $function$select $1;$function$
    LANGUAGE SQL IMMUTABLE RETURNS NULL ON NULL INPUT;
end
$$
```

Result: "Bind parameter(s)" dialog is being opened to bind parameter $1.
This only happens when using dollar-quotes inside the function definition (because of nested dollar quotes?).

**Comment by p91paul** [comment]
there should be an ignore button in the bind dialog that allows you to run the script as-is

**Comment by MarcoBorchert** [comment]
fixed in 6.3.4
